configs/xenguest_arm64_defconfig

```diff
@@ -42,3 +42,36 @@ CONFIG_CMD_FAT=y
 # CONFIG_MMC is not set
 # CONFIG_REQUIRE_SERIAL_CONSOLE is not set
 CONFIG_DM_SERIAL=y
+# CONFIG_SYS_SDRAM_BASE is not set
```

Comment:
Thanks. I have (probably) no objections regarding commit diff, but I have comment about the commit message. It is unclear why you are doing those changes.  I want you to read section "Describe your changes": 

https://www.kernel.org/doc/html/latest/process/submitting-patches.html#describe-your-changes

Yes, this is the Linux manual, but it describes requirements that are used in virtually all embedded open source projects. You can ignore the Linux-related parts, but please stick to all other rules. 

For this particular commit message, you also need to drop "pvblock" word, as it is misleading: there is nothing pvblock-specific in this commit. Also, it is not clear what exactly you are trying to accomplish, so it is hard to review this change properly.

Reply:
Hi @lorc,

After the review of what was upstreamed, I've figured out, that I need no any of those flags at all. This patch of Andrii was upstreamed, and then further patches in the upstream moved almost everything to:

- KConfig default values
- xenguest_arm64_defconfig ( yes, some of my added flags were already there )

Done.

P.S. I will review the Android-specific flags from the same perspective. Those flags which would be automatically added through KConfig are not needed to be presented in the defconfig.

Reply:
The same flags for xenguest_android_arm64_defconfig were reviewed and changed.